Bootloader/firmware/src/bootloader_usb_descriptors.c

```diff
@@ -223,22 +224,16 @@ static const USB_DEVICE_MASTER_DESCRIPTOR usbMasterDescriptor = {
   NULL,  // Pointer to high speed dev qualifier.
 };
 
-// Endpoint Table needed by the Device Layer.
-// ----------------------------------------------------------------------------
-static uint8_t __attribute__((aligned(512)))
-    g_endpoint_table[USB_DEVICE_ENDPOINT_TABLE_SIZE];
-
 // USB Device Layer Initialization Data
 // ----------------------------------------------------------------------------
 const USB_DEVICE_INIT usbDevInitData = {
   .moduleInit = {SYS_MODULE_POWER_RUN_FULL},
-  .stopInIdle = false,
-  .suspendInSleep = false,
-  .endpointTable = g_endpoint_table,
   .registeredFuncCount = 1,
   .registeredFunctions = (USB_DEVICE_FUNCTION_REGISTRATION_TABLE*) g_func_table,
   .usbMasterDescriptor = (USB_DEVICE_MASTER_DESCRIPTOR*) &usbMasterDescriptor,
   .deviceSpeed = USB_SPEED_FULL,
+  .driverIndex = DRV_USBFS_INDEX_0,
```

Comment:
The name is wrong (chalk that up to another harmony oddness). USBFS is actually the low level USB driver. not a filesystem as one would imagine.